### DISCUSSION TITLE
访问页面404，返回的error为504

### DISCUSSION THREAD

**Discussion by zylunx** [spencerwooo/onedrive-vercel-index#628] [Q&A] [answered]
```
{"error":{"code":"504","message":"An error occurred with your deployment"}}
```
![image.png](https://s2.loli.net/2022/05/01/2lFRiMVnS7qIA1t.png)

**Comment by myl7** [comment]
方便提供网站来进行测试吗？仅从此报错信息不太能发现问题。

**Reply by zylunx** [reply]
谢谢，已经找到问题，在连接到redis这一步有问题，现在已经解决
![image](https://user-images.githubusercontent.com/64573311/172831341-ce8e40db-3ead-4338-a352-d831c18df604.png)

**Comment by superen7527** [comment, marked as answer]
我改了一下标题就好了。可能是数据库成立还没对接 那个时候对接8分钟。我github改了title刷新是第十分钟。然后就可以了，不过和title应该没关系，可能是太快了等一会就好
